code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -155,6 +155,12 @@
 	pH = 11
 	value = REAGENT_VALUE_COMMON
 
+// Healing
+/datum/reagent/medicine/cryoxadone/on_hydroponics_apply(obj/item/seeds/myseed, datum/reagents/chems, obj/machinery/hydroponics/mytray, mob/user)
+	. = ..()
+	mytray.adjustHealth(round(chems.get_reagent_amount(type) * 3))
+	mytray.adjustToxic(-round(chems.get_reagent_amount(type) * 3))
```

Comment:
When you have 50u of cryoxadone in your beaker and transfer 5u of them to the tray, it will still heal 150 toxins and health